test/built-ins/Temporal/Instant/prototype/toZonedDateTimeISO/timezone-missing.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.tozoneddatetimeiso
+description: toZonedDateTimeISO() throws without parameter.
+includes: [temporalHelpers.js]
```

Comment:
Linter is complaining about this line, can just be removed.

Reply:
Fixed